content-src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -116,9 +118,26 @@ function groupSitesByDate(sites) {
     }
     groupedSites.get(day).push(site);
   }
+  groupedSites.forEach((value, key) => {
+    const sessions = groupSitesBySession(value);
+    groupedSites.set(key, sessions);
+  });
   return groupedSites;
 }
 
+function groupSitesBySession(sites) {
```

Comment:
Right now content uses the word `site` to mean the equivalent of link, filed https://github.com/mozilla/activity-streams/issues/374 to change that